index.html

```diff
@@ -0,0 +1,257 @@
+              </svg>
+            </a>
+          </div>
+          <div class="main-menu">
+            <ul>
+              <li><a href="./index.html"> Home </a></li>
+              <li class="last-child"><a href="./about-us.html"> About Us </a></li>
+            </ul>
+          </div>
+          <div class="menu-icon">
+            <svg height="20" width="20" xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+          </div>
+        </nav>
+      </div>
+      <div class="mobile-navigaton hide">
+        <div class="close-btn">
+          <svg height="20" width="20" xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+          </svg>
+        </div>
+        <div class="mobile-menu">
+          <ul>
+            <li> <a href="./index.html">Home</a></li>
+            <li> <a href="./about-us.html">About Us </a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section id="heading">
+      <div class="head head-about">
+        <div class="container">
+          <div class="head-content">
+            <h6>"Hello! Touring World"</h6>
+            <h1>Tourism Events Global summit <span class="year"> 2015 </span></h1>
+
+            <div class="head-text">
+              <p>A joyful celebration believing in the value of openess and sharing, creating a postive change with people from over 80 countries is
+                taking place in october, in Uganda.
+              </p>
+            </div>
+
+            <div class="head-date">
+              <h4>2015.10.15(THU) ~ 16(FRI)</h4>
+              <p>@ National Uganda Museum, Kololo and more.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </header>
+
+  <!-- Main Program section -->
+
+  <section id="main-program">
+    <div class="main-section">
+      <div class="container">
+        <div class="main-content">
+          <h3>Main Program</h3>
+          <div class="line">&nbsp;</div>
+          <div class="cards">
+            <div class="card">
+              <div class="icon">
+                <svg xmlns="http://www.w3.org/2000/svg" height="25" width="25" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19 11a7 7 0 01-7 7m0 0a7 7 0 01-7-7m7 7v4m0 0H8m4 0h4m-4-8a3 3 0 01-3-3V5a3 3 0 116 0v6a3 3 0 01-3 3z" />
+                </svg>
+              </div>
```

Comment:
- [ ] Hi these cards are designed well, although please make sure you properly align them vertically currently they are not centered in the middle vertically. If you look at the reference they should be.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/147154709-4c05a87c-f6e6-4f15-b0a4-7996467066e0.png)

## Reference Design
![image](https://user-images.githubusercontent.com/18730106/147154766-87f9c51c-2cc6-48c4-99e5-16b1bf5036f4.png)